lib/msf/core/db_manager/cred.rb

```diff
@@ -137,7 +137,7 @@ def report_auth_info(opts={})
 
     # Service management; assume the user knows what
     # he's talking about.
-    service = opts[:service] || report_service(:host => host, :port => port, :proto => proto, :name => sname, :workspace => wspace)
+    service = opts[:service] || msf_import_service(:host => host, :port => port, :proto => proto, :name => sname, :workspace => wspace)
```

Comment:
The normal db manager calls shouldn't be calling the `import` functions, which are for dealing with imports from files etc. Only the `imports` should be using the `import` functions; So this line should be reverted.